charts/astronomer/values.yaml

```diff
@@ -100,6 +100,17 @@ houston:
   jwtSigningKeySecretName: ~
   jwtSigningCertificateSecretName: ~
 
+  serviceAccount:
+    # Specifies whether a service account should be created
+    create: false
+    # Annotations to add to the service account
+    annotations: {}
+    # The name of the service account to use.
+    # If not set and create is true, a name is generated using the fullname template
+    name: ""
+    # Specifies whether a scc privilege should be created for custom sa
+    sccEnabled: false
```

Comment:
sccEnabled is particularly for openshift and we required that for registry for other services its handled in a different way so this is not required here.

Reply:
@pgvishnuram can you make a code suggestion to better indicate what section is not required here?

Reply:
this is removed.